src/xkbcomp/scanner.c

```diff
@@ -5,9 +5,13 @@
 
 #include "config.h"
 
+#include <assert.h>
+
 #include "xkbcomp-priv.h"
 #include "parser-priv.h"
 
+#define DECIMAL_SEPARATOR '.'
```

Comment:
```suggestion
const char DECIMAL_SEPARATOR = '.';
```

Reply:
I *really* have to stop using macros for every global var! Fixed